pycaching/geocaching.py

```diff
@@ -289,7 +289,8 @@ def search(self, point, limit=float("inf")):
                 badge = row.find("svg", class_="badge")
                 c.found = "found" in str(badge) if badge is not None else False
                 c.favorites = row.find(attrs={"data-column": "FavoritePoint"}).text
-                c.state = not (row.get("class") and "disabled" in row.get("class"))
+                if not (row.get("class") and "disabled" in row.get("class")):
```

Comment:
Did you add a test for this as well?

Reply:
Now I'm trying to add tests for this as well, and I forgot there's some problem with the `Geocaching.search()` method:
- https://github.com/tomasbedrich/pycaching/issues/182

Reply:
Now I'm trying to add tests for this as well, and I forgot there's some problem with the `Geocaching.search()` method:
- https://github.com/tomasbedrich/pycaching/issues/182

That means I can't record cassettes...

Reply:
Sorry, I did not remember this. In this case, we probably have to omit the corresponding test case here.